molecule/default/tests/test_default.py

```diff
@@ -25,3 +26,17 @@ def test_packages(host):
     installed = [package.is_installed for package in packages]
     assert len(pkgs) != 0
     assert all(installed)
+
+
+@pytest.mark.parametrize(
+    "f",
+    [
+        "/home/vnc/.config/autostart/light-locker.desktop",
+        "/home/vnc/.config/autostart/xfce4-power-manager-disable-dpms.desktop",
+        "/home/vnc/.config/autostart/xfce4-screensaver-disable.desktop",
+        "/home/vnc/.config/autostart/xfce4-screensaver-disable-screen-locking.desktop",
+    ],
+)
+def test_autostart_files_exist(host, f):
+    """Test that the file exists."""
+    assert host.file(f).exists
```

Comment:
Just for my own reference do we care about testing file permissions? We've been kind of on and off on this so I'm just looking to reach a consistent opinion.

Reply:
I have no strong opinions either way here.

Reply:
I added tests for the ownership and mode in commit ca2882a.